proposals/bluetooth.md

```diff
@@ -0,0 +1,67 @@
+# [Bluetooth](https://www.bluetooth.com/) Protocol Implementation
+
+## Goal
+Agent implements the bluetooth protocol for communicating with IOT devices.
```

Comment:
I would change this to: "To create a Discovery Handler that implements the bluetooth protocol for discovering IoT devices". While DHs can be embedded in the Agent the [implementation is a discovery handler](https://docs.akri.sh/development/handler-development)